spec/queries/atmosphere/appliances_affected_by_pmt_spec.rb

```diff
@@ -30,10 +32,10 @@
 
   context 'in development mode' do
     it 'finds affected appliances using dev_mode_property_set relation' do
-      dev_as = create(:dev_appliance_set)
-      appl1 = create(:appliance, appliance_type: at, appliance_set: dev_as)
-      create(:appliance, appliance_type: at)
+      appl1 = dev_appliance
       dev_pmt = appl1.dev_mode_property_set.port_mapping_templates.first
+      # other appliances which should not be taken into account
+      create(:appliance, appliance_type: at)
 
       affected_appl = Atmosphere::AppliancesAffectedByPmt.new(dev_pmt).find
```

Comment:
This looks funky. Wouldn't it make more sense to implement `find` as a static method of `AppliancesAffectedByPmt` (possibly renaming the class to something like `AffectedAppliancesLocalizer`) and parameterize this method with `dev_pmt`?

ATM this invocation reads like you're "creating" some sort of "appliance affected by PMT" and then invoking something on it. I understand what's really going on, but the choice of names is rather unfortunate...